core/error_api.php

```diff
@@ -480,30 +488,47 @@ function error_print_context( array $p_context ) {
 }
 
 /**
- * Print out a stack trace
+ * Get the stack trace as a string that can be logged or echoed to CLI output.
+ *
+ * @param Exception|null $p_exception The exception to print stack trace for.  Null will check last seen exception.
+ * @return string multi-line printout of stack trace.
  */
-function error_print_stack_trace() {
-	$t_stack = error_stack_trace();
+function error_stack_track_as_string( $p_exception = null ) {
```

Comment:
Should be *error_stack_trac**e**_as_string*, no ?

Reply:
Fixed typo.